buildSrc/src/main/java/org/opensearch/gradle/precommit/FilePermissionsTask.java

```diff
@@ -102,6 +100,11 @@ public FileCollection getFiles() {
 
     @TaskAction
     public void checkInvalidPermissions() throws IOException {
+        if (getFiles().isEmpty()) {
```

Comment:
Ah you solved it like that. Wa smy idea for forbiddenapis, too. But that very unelegant. I'd add the other annotation here.

Reply:
Yep, I will try your suggestion, it is cleaner for sure